Library/Formula/truncate.rb

```diff
@@ -0,0 +1,39 @@
+class Truncate < Formula
+  desc "truncates a file to a given size"
+  homepage "http://www.vanheusden.com/truncate"
+  url "http://www.vanheusden.com/truncate/truncate-0.8.tgz"
+  sha256 "165914d643351324d69096427d3ce5d3ab13aed3f7c990a71ba3bd0c6f278f00"
+
+  patch :DATA # remove missing file; rename syscalls for OSX which are natively 64bit
```

Comment:
Thanks, @flok99 .

I find you have change most xxx64 keywords to xxx. However, the first `off64_t get_file_size` still use `off64_t`, which leads to an error under MacOSX10.11.sdk.  
Would you change it to `off_t` ? As it already is `off_t old_size = get_file_size_fd(fd);` there.